rest/admin_api.go

```diff
@@ -173,6 +173,16 @@ func (h *handler) handleReplicate() error {
 		return err
 	}
 
+	//If source and/or target are local DB names add local AdminInterface URL
+	localDbUrl := "http://" + *h.server.config.AdminInterface
```

Comment:
Yes

Set the following in config.json

`"adminInterface":":4985",`

Trigger replication with the following curl:

```
curl -X POST http://localhost:4985/_replicate -H "Content-Type: application/json" -d '{"source":"source", "target":"target"}'
```

Results in log output from SG of:

```
2016-04-05T08:55:15.795+01:00 HTTP:  #011: POST /_replicate  (ADMIN)
2016-04-05T08:55:15.795+01:00 Replicate: Starting replication with parameters {ReplicationId: Source:http://:4985 SourceDb:source Channels:[] Target:http://:4985 TargetDb:target ChangesFeedLimit:0 Lifecycle:0 Disabled:false Async:false}
2016-04-05T08:55:15.796+01:00 HTTP:  #012: GET /target/_local/da39a3ee5e6b4b0d3255bfef95601890afd80709  (ADMIN)
2016-04-05T08:55:15.796+01:00 HTTP: #012:     --> 404 missing  (0.2 ms)
2016-04-05T08:55:15.797+01:00 HTTP:  #013: GET /source/_changes?feed=normal&limit=50&heartbeat=30000&style=all_docs&since=0  (ADMIN)
2016-04-05T08:55:15.797+01:00 Changes+: Int sequence multi changes feed...
2016-04-05T08:55:15.797+01:00 Changes: MultiChangesFeed({*}, {Since:0 Limit:50 Conflicts:true IncludeDocs:false Wait:false Continuous:false Terminator:0xc82042a360 HeartbeatMs:30000 TimeoutMs:300000 ActiveOnly:false}) ... 
2016-04-05T08:55:15.797+01:00 Changes+: MultiChangesFeed: channels expand to channels.TimedSet{"*":channels.VbSequence{VbNo:(*uint16)(nil), Sequence:0x0}} ... 
2016-04-05T08:55:15.797+01:00 Cache: getCachedChanges("*", {1 0 0 0 <nil> <nil> %!d(string=) 0 0 0 %!d(string=)}) --> 0 changes valid from #9
2016-04-05T08:55:15.797+01:00 Cache:   Querying 'channels' view for "*" (start=#1, end=#9, limit=50)
2016-04-05T08:55:15.800+01:00 Cache:     Got 4 rows from view for "*": #4 ... #8
2016-04-05T08:55:15.800+01:00 Cache: GetChangesInChannel("*") --> 4 rows
2016-04-05T08:55:15.800+01:00 DIndex+: [changesFeed] Found 4 changes for channel *
2016-04-05T08:55:15.800+01:00 Changes+: Sending seq:4 from channel *
2016-04-05T08:55:15.800+01:00 Changes+: Sending seq:6 from channel *
2016-04-05T08:55:15.800+01:00 Changes+: MultiChangesFeed sending &{Seq:4 ID:doc1 Deleted:true Removed:{} Doc:map[] Changes:[map[rev:3-3ccdc2bf063e2102c2aa9dd447f639a0]] Err:<nil> allRemoved:false branched:false} 
2016-04-05T08:55:15.800+01:00 Changes+: Sending seq:7 from channel *
2016-04-05T08:55:15.800+01:00 Changes+: MultiChangesFeed sending &{Seq:6 ID:doc2 Deleted:false Removed:{} Doc:map[] Changes:[map[rev:2-a96aff0b11b431af609db88acd01eb84]] Err:<nil> allRemoved:false branched:false} 
2016-04-05T08:55:15.800+01:00 Changes+: Sending seq:8 from channel *
2016-04-05T08:55:15.800+01:00 Changes+: MultiChangesFeed sending &{Seq:7 ID:749946e5a7c098e1de289321c4f9aadd Deleted:false Removed:{} Doc:map[] Changes:[map[rev:1-cd809becc169215072fd567eebd8b8de]] Err:<nil> allRemoved:false branched:false} 
2016-04-05T08:55:15.800+01:00 Changes+: MultiChangesFeed sending &{Seq:8 ID:b97e5042fc40a583524c56eb19574fa4 Deleted:false Removed:{} Doc:map[] Changes:[map[rev:1-98f2786e47003c499b8f4b7fc65bd192]] Err:<nil> allRemoved:false branched:false} 
2016-04-05T08:55:15.800+01:00 Changes: MultiChangesFeed done 
2016-04-05T08:55:15.800+01:00 HTTP+: #013:     --> 200 OK  (0.0 ms)
2016-04-05T08:55:15.801+01:00 HTTP:  #014: POST /target/_revs_diff  (ADMIN)
2016-04-05T08:55:15.803+01:00 HTTP+: #014:     --> 200   (2.2 ms)
2016-04-05T08:55:15.803+01:00 HTTP:  #015: PUT /target/_local/da39a3ee5e6b4b0d3255bfef95601890afd80709  (ADMIN)
2016-04-05T08:55:15.804+01:00 HTTP+: #015:     --> 201   (0.3 ms)
2016-04-05T08:55:15.804+01:00 HTTP:  #016: GET /target/_local/da39a3ee5e6b4b0d3255bfef95601890afd80709  (ADMIN)
2016-04-05T08:55:15.804+01:00 HTTP+: #016:     --> 200   (0.1 ms)
2016-04-05T08:55:15.804+01:00 HTTP:  #017: GET /source/_changes?feed=normal&limit=50&heartbeat=30000&style=all_docs&since=8  (ADMIN)
2016-04-05T08:55:15.804+01:00 Changes+: Int sequence multi changes feed...
2016-04-05T08:55:15.804+01:00 Changes: MultiChangesFeed({*}, {Since:8 Limit:50 Conflicts:true IncludeDocs:false Wait:false Continuous:false Terminator:0xc8203ea2a0 HeartbeatMs:30000 TimeoutMs:300000 ActiveOnly:false}) ... 
2016-04-05T08:55:15.804+01:00 Changes+: MultiChangesFeed: channels expand to channels.TimedSet{"*":channels.VbSequence{VbNo:(*uint16)(nil), Sequence:0x0}} ... 
2016-04-05T08:55:15.804+01:00 Cache: getCachedChanges("*", {1 0 0 8 <nil> <nil> %!d(string=) 0 0 0 %!d(string=)}) --> 0 changes valid from #9
2016-04-05T08:55:15.804+01:00 DIndex+: [changesFeed] Found 0 changes for channel *
2016-04-05T08:55:15.804+01:00 Changes: MultiChangesFeed done 
2016-04-05T08:55:15.804+01:00 HTTP+: #017:     --> 200 OK  (0.0 ms)
2016-04-05T08:55:15.805+01:00 HTTP+: #011:     --> 200   (9.2 ms)
```